go-install/linux.md

```diff
@@ -1,21 +1,49 @@
-# Setup instructions for Linux
+# Personal Linux setup instructions
 
-1. Run the following commands in terminal:
+## Setup instructions for Ubuntu 20.04 LTS (focal)
+
+1. Run the following commands in terminal. If bash is not your default shell, subsitute `.bashrc` for the appropriate file (e.g. `.zshrc` for zsh).
+
+```bash
+sudo apt-get update
+sudo apt-get install golang=2:1.13~1ubuntu2 --yes
+echo 'export GOPATH="$HOME/go"' >> ~/.bashrc
+echo 'export PATH="$PATH:$GOPATH/bin"' >> ~/.bashrc
+```
+
+2. Verify that `go version` shows a go installation of version `>=1.13`.
```

Comment:
will need to `source ~/.bashrc` or re-open terminal

Reply:
Done